### PR TITLE
Add docs for task count include filter

### PR DESCRIPTION
**Pull request by JeffBain**
Refs SDE-2314

### CODE REVIEW

**Comment by funkaoshi** [comment]
+1

File: source/includes/tasks.md [hunk 1]
```diff
@@ -177,6 +194,7 @@ problem             | Includes the problem that the task is related to
 related             | Includes a list of related tasks
 tags                | Includes a list of tags associated to the task
 regulation_sections | Includes a list of regulation sections to which this task belongs.
+task_counts         | Includes counts of tasks broken down by priority and completeness for the project
```
Comment (RouganStriker) [inline source/includes/tasks.md]: Should we include the priority mapping in the description?
ie. high(7-10), medium(4-6), low(1-3)

**Comment by jnlantos** [comment]
+1 (I agree with @RouganStriker that adding the priority mapping might be helpful.)

**Comment by geoffwhittington** [comment]
The corresponding story has passed QA, should this get merged then?

**Comment by sibartlett** [comment]
I just noticed that this pull request is modifying the wrong file. The task_counts include is part of the Project endpoint, not Task endpoint.

**Comment by sibartlett** [comment]
I've fixed and merged this PR.

**Pull request merged**